.gitignore

```diff
@@ -15,5 +15,3 @@ logs/
 # Intellij #
 *.iml
 .idea/
-
-logs/
```

Comment:
Nope, if you look above, "logs" are already present in .gitignore file. This line was just a duplicate